guide/payments/units-and-symbols.md

```diff
@@ -0,0 +1,147 @@
+amount of different notation standards around the world, it focuses on bitcoin.
+
+Illustration sources
+
+https://www.figma.com/file/VB3GQdAnhl8yta44DY3PSV/Bitcoin-Wallet-UI-Kit?node-id=2318%3A61126
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/payments/units-and-symbols/units-and-symbols.png"
+   retina = "/assets/images/guide/payments/units-and-symbols/units-and-symbols@2x.png"
+   mobile = "/assets/images/guide/payments/units-and-symbols/units-and-symbols-mobile.png"
+   mobileRetina = "/assets/images/guide/payments/units-and-symbols/units-and-symbols-mobile@2x.png"
+   alt-text = "Bitcoin value with annotations for parts of the number display."
+   caption = "Aspects to consider when formatting bitcoin amounts."
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+# Units and symbols
+
+Bitcoin, bits, sats. The format and presentation of bitcoin values are probably amongst the most challenging for new users. Here we provide observations and recommendations on current usage in the context of established, global standards. The surest way to achieve wide-scale adoption is to formalize what is already being used today.
+
+Despite the challenge of the fractional aspect and the abundance of digits used to present the value of bitcoin, the intent is to serve as a guide to format the values so they can easily be scanned, quick to read, and intuitive to remember.
+
+#### Current adoption
+
+The most common way bitcoin is expressed is as BTC (bitcoin) or sat (satoshi), with 1 bitcoin being 100 000 000 satoshi. While not as common, denominations of BTC such as mBTC ("millibitcoins"), μBTC ("bits") are also sometimes used, as well as millisatoshi (msat) in the Lightning network. The unicode symbol ₿, formalized in June 2017, can also be used to represent BTC or bitcoin, but few typefaces support at the time of this writing. The chart below illustrates how each unit relates to the bitcoin unit.
+
+| Unit         | Symbol       | Bitcoin value     |
+| ------------ | ------------ | ----------------- |
+| bitcoin      | BTC or ₿     | 1.00 000 000      |
+| millibit     | mBTC         | 0.00 100 000      |
+| bit          | μBTC         | 0.00 000 100      |
+| satoshi      | sat          | 0.00 000 001      |
+| millisatoshi | msat         | 0.00 000 000 001  |
+
+For more information, see the Bitcoin Wiki:
+
+- [Units](https://en.bitcoin.it/wiki/Units)
+- [Satoshi](https://en.bitcoin.it/wiki/Satoshi_(unit))
+
+#### Recommended interaction
+
+When displaying bitcoin values, the default unit for on-chain wallets should be bitcoin with 8 decimal places, and satosho for Lightning wallets. Due to the challenging nature of scanning a monetary value with more than 2 decimal places, the user should be given the option to choose their preferred format across the application (for example, in the settings section) as well as contextually, whenever the value is primarily displayed.
```

Comment:
```suggestion
When displaying bitcoin values, the default unit should be bitcoin for on-chain wallets, and satoshi for Lightning wallets. Due to the challenging nature of scanning a monetary value with more than 2 decimal places, the user should be given the option to choose their preferred format across the application (for example, in the settings) as well as contextually, whenever a value is displayed.
```

Reply:
What's your take on 8 decimals? Always show them (1.23000000) or trim to 2-4 digits (1.23) as contextually appropriate?

Reply:
I say always trim, trailing zeros that don't fill a need always make it harder to parse imo.

Reply:
I usually trim as well, but I find figuring out the satoshi amount is more difficult when they're not there.

Reply:
just realised even with dollars and euros we tend to not trim 0s unless its a whole amount.

$1
$1.00
$1.10

Reply:
```suggestion
When displaying bitcoin values, the default unit for on-chain wallets should be bitcoin with 8 decimal places, and satoshi for Lightning wallets. Due to the challenging nature of scanning amounts with more than 2 decimal places, the user should be given the option to choose their preferred format across the application (for example, in the settings section) as well as contextually, whenever the value is primarily displayed.
```

Reply:
> the user should be given the option to choose their preferred format across the application (for example, in the application's settings) as well as contextually, whenever the value is primarily displayed

I guess the pattern this is describing is on an amount input, or wallet overview screen. How does this apply on payment history screen though?

Reply:
Payment history would be driven by the app settings. Assumption is that the user generally has one way they like to see things, which drives the default across the app.

While entering an amount, you could be in a unique situation where you want to pay someone in a different currency, or an extremely low or high amount, that is different than your "normal". That's when you want that option to switch the format.

Having that option on the main balance on the home screen is just a minor convenience thing, IMHO.